src/Image.php

```diff
@@ -73,6 +74,22 @@ public function background(Background $background, float $opacity = 1.0): self
         return $this;
     }
 
+    public function backgroundURL(string $backgroundURL, float $opacity = 1.0): self
```

Comment:
I prefer true camelCase for this
```suggestion
    public function backgroundUrl(string $backgroundUrl, float $opacity = 1.0): self
```